OracleDatabase/SingleInstance/dockerfiles/21.3.0/Dockerfile

```diff
@@ -36,14 +36,15 @@ LABEL "provider"="Oracle"                                               \
 
 # Argument to control removal of components not needed after db software installation
 ARG SLIMMING=true
+ARG INSTALL_FILE_1="LINUX.X64_213000_db_home.zip"
 
 # Environment variables required for this build (do NOT change)
 # -------------------------------------------------------------
 ENV ORACLE_BASE=/opt/oracle \
     ORACLE_HOME=/opt/oracle/product/21c/dbhome_1 \
     ORACLE_BASE_HOME=/opt/oracle/homes/OraDB21Home1 \
     INSTALL_DIR=/opt/install \
-    INSTALL_FILE_1="LINUX.X64_213000_db_home.zip" \
```

Comment:
I don't think this line is necessary anymore, will the `ARG INSTALL_FILE_1` be handled like any other environment variable?

Reply:
In this Dockerfile, we have three stages. The scope of ARG ends with a build-stage ([ref documentation](https://docs.docker.com/engine/reference/builder/#scope)). We will need to replicate this ARG line i.e. `ARG INSTALL_FILE_1="LINUX.X64_193000_db_home.zip"` to each of those build stages. Using ENV mitigate that issue.

Reply:
Ah, I see, thanks a lot for the clarification!